### PR TITLE
[13.0][ADD] project_administrator_restricted_visibility: new module

### PR DESCRIPTION
**Pull request by ernestotejeda**
TT42466

Add a module that extends the functionality of 'Project' module to add a new 'Restricted Project Administrator' access group.  Users with that group will be like 'Project Administrators' but with the same restrictions (ir.rule) on project records as 'Project Users' access group.

### CODE REVIEW

**Comment by ernestotejeda** [comment]
ping @pedrobaeza

File: project_administrator_restricted_visibility/README.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Settings > Users & Companies > Users*
+#. Create a user or edit an existing. If the user is internal,
+   he/she will be able to see a new access group called
```
Comment (pedrobaeza) [inline project_administrator_restricted_visibility/README.rst]:
```suggestion
#. Create a user or edit an existing.
#. A new access group called ...
```

File: project_administrator_restricted_visibility/README.rst [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+   'Restricted Project Administrator' under the 'Projects' category.
+   If this group is selected, the user will have the same access right
+   as the 'Project Administrator' but with the project visibility
+   of the 'Project User'."
```
Comment (pedrobaeza) [inline project_administrator_restricted_visibility/README.rst]:
```suggestion
Usage
=====

The new group has the same access rights as the administrator (and see the same menus), but the project visibility is restricted as a project user.
```

File: project_administrator_restricted_visibility/security/project_security.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    <record id="group_full_project_manager" model="res.groups">
+        <field name="name">Project Administrator</field>
+        <field name="category_id" ref="base.module_category_operations_project" />
+        <field name="implied_ids" eval="[(4, ref('project.group_project_manager'))]" />
+    </record>
+    <!-- Rename old 'Project Administrator' access group name -->
+    <record id="project.group_project_manager" model="res.groups">
+        <field name="name">Restricted Project Administrator</field>
+    </record>
+    <data noupdate="1">
```
Comment (pedrobaeza) [inline project_administrator_restricted_visibility/security/project_security.xml]: AFAIK, you don't need to put it in noupdate section.

**Review by pedrobaeza — COMMENTED** [review]
Put an uninstall hook that leaves data as it was before installing the module.

**Comment by ernestotejeda** [comment]
@pedrobaeza could please you review again?

**Review by pedrobaeza — APPROVED** [review]

**Review by victoralmau — APPROVED** [review]
Code review OK

**Pull request merged**